src/libs/actions/IOU.js

```diff
@@ -720,11 +720,152 @@ function payIOUReport({
     return promiseWithHandlers;
 }
 
+/**
+ * @param {String} chatReportID - Report ID of the chat where the IOU is.
+ * @param {Number} amount - IOU amount in cents.
+ * @param {String} currency - IOU currency.
+ * @param {String} comment
+ * @param {Array} participants - Participants on the IOU
+ */
+function sendMoneyWithWallet(chatReportID, amount, currency, comment, participants) {
```

Comment:
We will need to add another report action here, that is essentially the same as `requestMoney` [here](https://github.com/Expensify/App/blob/2356010ff7817ae27e603558b6905474929465d6/src/libs/actions/IOU.js#L56-L89) and then we add the PAY action

Reply:
I think I already do a bit further down no? Or am I missing something?

Reply:
Sorry @sketchydroide this comment was wrong, there is just one PAY report Action when sending money. I have thought the entire time it is compounded of two but it is just one with the IOUDetails